app/src/main/java/com/jvoye/tasky/agenda/domain/AgendaRepository.kt

```diff
@@ -0,0 +1,5 @@
+package com.jvoye.tasky.agenda.domain
+
+interface AgendaRepository {
+    suspend fun getAgendaItems(): List<AgendaItem>
```

Comment:
For a follow-up PR 😊: since persistence is part of the requirements, it’ll be easier to work in a reactive manner. Consider exposing repository results as Flow so UI can observe changes from the DB/cache automatically (and we avoid manual refresh wiring).
```kotlin
interface AgendaRepository {
    fun observeAgendaItems(): Flow<List<AgendaItem>>
 }
```